frontend/projects/upgrade/src/app/features/dashboard/experiment-users/components/stratification-factor-list/stratification-factor-list.component.scss

```diff
@@ -0,0 +1,91 @@
+@import '../../common/table-view.scss';
+
+.stratification-factors-list-container{
+  border-radius: 10px;
+  box-shadow: unset !important;
+  padding: 0;
+  height: 100%;
+
+  .header {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+
+    button {
```

Comment:
Just a note here that using a native element tag as a css selector is asking for trouble, can a class be used instead? I know it's happening in many places in codebase but it's an easy thing to do to avoid very annoying css debugging behavior.